Modules/RaidManager/Raid.lua

```diff
@@ -6,14 +6,15 @@ local CONSTANTS = CLM.CONSTANTS
 local UTILS     = CLM.UTILS
 -- ------------------------------- --
 
-local setmetatable, ipairs, tinsert, tsort = setmetatable, ipairs, table.insert, table.sort
+local setmetatable, ipairs, tsort = setmetatable, ipairs, table.sort
 
 local Raid = {} -- Raid information
+
+Raid.__index = Raid
```

Comment:
What is the difference between this line and `self.__index = self`?

Reply:
They are the same; but I moved it outside the constructor so that it's called once instead of for every new raid model created.
(And outside the function there is no `self`)